traffic_monitor/cache/astats_test.go

```diff
@@ -60,14 +57,11 @@ func TestAstatsCSV(t *testing.T) {
 	pl := &poller.HTTPPollCtx{HTTPHeader: http.Header{}}
 	ctx := interface{}(pl)
 	ctx.(*poller.HTTPPollCtx).HTTPHeader.Set("Content-Type", "text/csv")
-	_, thismap, err := astatsParse("testCache", file, ctx)
+	_, _, err = astatsParse("testCache", file, ctx)
```

Comment:
We still want to verify the correct size of the map, right?

Reply:
its more important that it just doesnt crash. There are already differences between both the json and csv outputs because the CSV one will strip out extraneous fields. Just verifying the size doesnt mean much since you could throw some random data into the files and still have it be allocated in to a proper map

This is also all the original test was doing as well, only checking that it didnt crash, beyond printing a value that wasnt checked or useful for anything, however it was only checking the jsoniter unmarshal capability and not checking that the TM functions themselves didnt cause issues

Reply:
That being said I could add some value checks in the map just to test for existence, but Id prefer to do that in the next PR. I plan on working on a follow up directly after this is merged to add similar support to stats_over_http, but that is dependent on this baseline being in since it needs the Accept support and changes to allow the contexts to be passed around